iup/eval_projections.py

```diff
@@ -1,5 +1,100 @@
+import polars as pl
 import altair as alt
 
+# get mspe #
+def get_mspe(
```

Comment:
This function is duplicated across these two files? It should be in only one place

Reply:
See [issue #57](https://github.com/CDCgov/cfa-immunization-uptake-projection/issues/57)